### DISCUSSION TITLE
Submitting an Amazon Advertising Partner application fails

### DISCUSSION THREAD

**Discussion by jiwei3187553** [amzn/ads-advanced-tools-docs#67] [Q&A]
I'm trying to submit the application form for Amazon Advertising Partner, and I have all the information filled out, but there is an error when I submit it.

Please ensure that you are signed into the Amazon account affiliated with your corporate email address. If you do not have an Amazon Account affiliated with your corporate email address, you may [update your current Amazon account or create an account ](https://www.amazon.com/your-account).

![img2](https://user-images.githubusercontent.com/25273864/229761101-a196675b-06e6-43ec-a636-fbb65cfb4d6b.png)

How can I submit the form correctly, I have opened the cass many times, but the reply I get is that they are not sure

**Comment by jerirose** [comment]
Hi @jiwei3187553 -- as discussed in #51, for issues registering with the Partner Network, consult the [Partner Network help center content on registration](https://advertising.amazon.com/partner-network/help/GEBCGKXA5SGBSQ59). 

Joining the Partner Network is one way to gain access to the API, but it is not required. Depending on your type of business, you might instead choose to [apply for API access as a direct advertiser](https://advertising.amazon.com/API/docs/en-us/onboarding/apply-for-access).

**Reply by jiwei3187553** [reply]
@jerirose Thank you for your reply, the [Partner Network help center content on registration](https://advertising.amazon.com/partner-network/help/GEBCGKXA5SGBSQ59), there is the only solution to the problem is the bottom of the contact us, but click on the contact us link will also report an error, resulting in a dead end, I registered myself prompted email problems, click help tell me to use the correct email, there is no team can deal with this problem, or tell me what to do to be able to, we are an advertising software company, so can only join the partners, I am not sure if the application submitted as an advertiser can get third-party business advertising authorization.
![img4](https://user-images.githubusercontent.com/25273864/230249439-fa213a05-6bfe-4b43-9db4-7a6d78055f08.png)

![img3](https://user-images.githubusercontent.com/25273864/230249114-889b913d-66ce-4609-810c-67cdac3d9dda.png)

**Comment by AdsAPIPM** [comment]
Hi. Have you seen this FAQ in the Partner Network[ help page](https://advertising.amazon.com/partner-network/help/GEBCGKXA5SGBSQ59)

Which Amazon account should I use to register my business on the Amazon Ads Partner Network?

To register your business on the Partner Network, we require that you use an Amazon account that is associated with your business email address. We do not recommend using a shared inbox to register for the Partner Network. We suggest signing in with the existing Amazon account credentials you use to support advertisers, but if you do not have an Amazon account, you will need to create one. If you have an existing manager account, we recommend using it to register on Partner Network, then add the users and advertisers that are already linked to the Manager account. **Please keep in mind, the email address domain used to register will need to have a direct association with your company name.**
